src/components/RequestsTable/RequestsTable.tsx

```diff
@@ -9,88 +9,97 @@ import {
   DataGridHeaderCell,
   DataGridProps,
   DataGridRow,
-  Field,
-  Input,
-  Popover,
-  PopoverSurface,
-  PopoverTrigger,
-  Radio,
-  RadioGroup,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
   Spinner,
   TableCellLayout,
   TableColumnDefinition,
+  TableColumnId,
+  TableColumnSizingOptions,
   createTableColumn,
 } from "@fluentui/react-components";
-import { ArrowNextRegular, ArrowPreviousRegular } from "@fluentui/react-icons";
-import { useState } from "react";
+import {
+  ArrowNextRegular,
+  ArrowPreviousRegular,
+  FilterRegular,
+} from "@fluentui/react-icons";
+import { useCallback, useRef, useState } from "react";
 import { Link } from "react-router-dom";
-import { FilterRegular } from "@fluentui/react-icons/lib/fonts";
-import { SortDownIcon, SortUpIcon } from "@fluentui/react-icons-mdl2";
+import { FilterIcon } from "@fluentui/react-icons-mdl2";
+import FilterRequestsDrawer from "./FilterRequests";
 
-const columnSizingOptions = {
-  positionTitle: {
-    minWidth: 80,
-    defaultWidth: 280,
-  },
-  requestType: {
-    minWidth: 80,
-    defaultWidth: 220,
+interface requestFilter {
+  column: string;
+  filter: string | Date | number;
+}
+
+const PositionTitle = createTableColumn<RPARequest>({
+  columnId: "positionTitle",
+  compare: (_a, _b) => 1, // Change nothing, but utilize table's sorting icons
+  renderHeaderCell: (filtered = false) => {
+    return <>Position Title {filtered && <FilterIcon />}</>;
   },
-  systemSeriesGrade: {
-    minWidth: 80,
-    defaultWidth: 100,
+  renderCell: (item) => {
+    return (
+      <TableCellLayout truncate>
+        <Link to={"/Request/" + item.Id}>{item.positionTitle}</Link>
+      </TableCellLayout>
+    );
   },
-  officeSymbol: {
-    minWidth: 80,
-    defaultWidth: 120,
+});
+
+const RequestType = createTableColumn<RPARequest>({
+  columnId: "requestType",
+  compare: (_a, _b) => 1, // Change nothing, but utilize table's sorting icons
```

Comment:
Why did you get rid of all the compare functions?  I assume it is because you plan on updating the RequestApi to do the sorting on the server, correct?